custom_components/easee/entity.py

```diff
@@ -30,22 +31,34 @@ def round_to_dec(value, decimals=None, unit=None):
         pass
     return value
 
+
 def round_2_dec(value, unit=None):
     return round_to_dec(value, 2, unit)
 
+
 def round_1_dec(value, unit=None):
     return round_to_dec(value, 1, unit)
 
+
 def round_0_dec(value, unit=None):
     return round_to_dec(value, None, unit)
 
+
+def map_ea_status(value, unit=None):
```

Comment:
map_charger_status is probably a better name

Reply:
OK